diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/Makefile.am libunwind-0.99-alpha/Makefile.am
--- libunwind-0.99-alpha-orig/Makefile.am	2006-07-27 05:13:14.000000000 +0200
+++ libunwind-0.99-alpha/Makefile.am	2008-02-24 06:43:08.000000000 +0100
@@ -40,6 +40,6 @@ EXTRA_DIST = include/dwarf.h include/dwa
 	include/libunwind-x86.h include/tdep-x86/libunwind_i.h		\
 	include/tdep-x86/jmpbuf.h include/tdep-x86/dwarf-config.h	\
 	include/libunwind-x86_64.h include/tdep-x86_64/libunwind_i.h	\
-	include/tdep-x86_64/jmpbuf.h include/tdep-x86_64/dwarf-config.h
-
-DISTCLEANFILES = include/libunwind.h include/tdep
+	include/tdep-x86_64/jmpbuf.h include/tdep-x86_64/dwarf-config.h	\
+	include/libunwind.h include/tdep/libunwind_i.h			\
+	include/tdep/jmpbuf.h include/tdep/dwarf-config.h
diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/configure.in libunwind-0.99-alpha/configure.in
--- libunwind-0.99-alpha-orig/configure.in	2008-02-24 06:29:08.000000000 +0100
+++ libunwind-0.99-alpha/configure.in	2008-02-24 06:30:29.000000000 +0100
@@ -80,8 +80,6 @@ AM_CONDITIONAL(OS_HPUX, expr x$target_os
 if test x$target_arch != x$build_arch; then
   CPPFLAGS="${CPPFLAGS} -DUNW_REMOTE_ONLY"
 fi
-AC_CONFIG_LINKS(include/libunwind.h:include/libunwind-$target_arch.h
-		include/tdep:include/tdep-$target_arch)
 
 AC_ARG_ENABLE(debug,
 [  --enable-debug          turn on debug support (slows down execution)],
diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/include/libunwind.h libunwind-0.99-alpha/include/libunwind.h
--- libunwind-0.99-alpha-orig/include/libunwind.h	1970-01-01 01:00:00.000000000 +0100
+++ libunwind-0.99-alpha/include/libunwind.h	2008-02-24 06:40:59.000000000 +0100
@@ -0,0 +1,17 @@
+/* Provide a real file - not a symlink - as it would cause multiarch conflicts
+   (when multiple different arch releases are installed simultaneously.
+   https://bugzilla.redhat.com/show_bug.cgi?id=342451  */
+
+#if defined __x86_64__
+# include "libunwind-x86_64.h"
+#elif defined __i386__
+# include "libunwind-x86.h"
+#elif defined __ia64__
+# include "libunwind-ia64.h"
+#elif defined __powerpc64__
+# include "libunwind-ppc64.h"
+#elif defined __hpux
+# include "libunwind-hppa.h"
+#else
+# error "Unsupported arch"
+#endif
diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/include/tdep/dwarf-config.h libunwind-0.99-alpha/include/tdep/dwarf-config.h
--- libunwind-0.99-alpha-orig/include/tdep/dwarf-config.h	1970-01-01 01:00:00.000000000 +0100
+++ libunwind-0.99-alpha/include/tdep/dwarf-config.h	2008-02-24 06:41:02.000000000 +0100
@@ -0,0 +1,17 @@
+/* Provide a real file - not a symlink - as it would cause multiarch conflicts
+   (when multiple different arch releases are installed simultaneously.
+   https://bugzilla.redhat.com/show_bug.cgi?id=342451  */
+
+#if defined __x86_64__
+# include "tdep-x86_64/dwarf-config.h"
+#elif defined __i386__
+# include "tdep-x86/dwarf-config.h"
+#elif defined __ia64__
+# include "tdep-ia64/dwarf-config.h"
+#elif defined __powerpc64__
+# include "tdep-ppc64/dwarf-config.h"
+#elif defined __hpux
+# include "tdep-hppa/dwarf-config.h"
+#else
+# error "Unsupported arch"
+#endif
diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/include/tdep/jmpbuf.h libunwind-0.99-alpha/include/tdep/jmpbuf.h
--- libunwind-0.99-alpha-orig/include/tdep/jmpbuf.h	1970-01-01 01:00:00.000000000 +0100
+++ libunwind-0.99-alpha/include/tdep/jmpbuf.h	2008-02-24 06:41:04.000000000 +0100
@@ -0,0 +1,17 @@
+/* Provide a real file - not a symlink - as it would cause multiarch conflicts
+   (when multiple different arch releases are installed simultaneously.
+   https://bugzilla.redhat.com/show_bug.cgi?id=342451  */
+
+#if defined __x86_64__
+# include "tdep-x86_64/jmpbuf.h"
+#elif defined __i386__
+# include "tdep-x86/jmpbuf.h"
+#elif defined __ia64__
+# include "tdep-ia64/jmpbuf.h"
+#elif defined __powerpc64__
+# include "tdep-ppc64/jmpbuf.h"
+#elif defined __hpux
+# include "tdep-hppa/jmpbuf.h"
+#else
+# error "Unsupported arch"
+#endif
diff -u -X /home/jkratoch/.diffi.list -ruNp -x Makefile -x Makefile.in libunwind-0.99-alpha-orig/include/tdep/libunwind_i.h libunwind-0.99-alpha/include/tdep/libunwind_i.h
--- libunwind-0.99-alpha-orig/include/tdep/libunwind_i.h	1970-01-01 01:00:00.000000000 +0100
+++ libunwind-0.99-alpha/include/tdep/libunwind_i.h	2008-02-24 06:41:06.000000000 +0100
@@ -0,0 +1,17 @@
+/* Provide a real file - not a symlink - as it would cause multiarch conflicts
+   (when multiple different arch releases are installed simultaneously.
+   https://bugzilla.redhat.com/show_bug.cgi?id=342451  */
+
+#if defined __x86_64__
+# include "tdep-x86_64/libunwind_i.h"
+#elif defined __i386__
+# include "tdep-x86/libunwind_i.h"
+#elif defined __ia64__
+# include "tdep-ia64/libunwind_i.h"
+#elif defined __powerpc64__
+# include "tdep-ppc64/libunwind_i.h"
+#elif defined __hpux
+# include "tdep-hppa/libunwind_i.h"
+#else
+# error "Unsupported arch"
+#endif
