diff -u -X /home/jkratoch/.diffi.list -rup libunwind-0.99-alpha-orig/tests/Makefile.am libunwind-0.99-alpha/tests/Makefile.am
--- libunwind-0.99-alpha-orig/tests/Makefile.am	2008-02-23 07:58:30.000000000 +0100
+++ libunwind-0.99-alpha/tests/Makefile.am	2008-02-23 18:59:09.000000000 +0100
@@ -4,7 +4,9 @@ EXTRA_DIST =	run-ia64-test-dyn1 run-ptra
 		run-check-namespace check-namespace.sh.in run-ptrace-signull 	\
 		run-ptrace-stepper
 
-noinst_PROGRAMS_common =
+noinst_PROGRAMS_common = orphanripper
+orphanripper_LDADD = -lutil
+TESTS_ENVIRONMENT = ./orphanripper -t 120
 check_PROGRAMS_common = test-proc-info test-static-link \
 			test-strerror
 check_SCRIPTS_common =	run-check-namespace
